### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by Hotanya**
Adding a SECURITY.md so any identified security issues can be reported and triaged.

### CODE REVIEW

**Comment by raon0211** [comment]
Thanks! We should add a security policy, but I guess we have to add the contents before adding this file.

**Comment by Hotanya** [comment]
Hey thanks for the prompt response. I've identified a security vulnerability in this project, once the SECURITY.md is setup, I can report it securely to you. You can take a look at https://github.com/projectdiscovery/nuclei/security for an example of the file contents.

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 5.1.x   | :white_check_mark: |
+| 5.0.x   | :x:                |
+| 4.0.x   | :white_check_mark: |
+| < 4.0   | :x:                |
```
Comment (raon0211) [inline SECURITY.md:13]:
```suggestion
The following table describes the versions of this project that are currently supported with security updates:

| Version | Supported          |
| ------- | ------------------ |
| 1.x   | :white_check_mark: |
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Reporting a Vulnerability
+
+Use this section to tell people how to report a vulnerability.
+
+Tell them where to go, how often they can expect to get an update on a
+reported vulnerability, what to expect if the vulnerability is accepted or
+declined, etc.
```
Comment (raon0211) [inline SECURITY.md:21]: We consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.

If you discover a security vulnerability, report the security issue to the project maintainers directly at [opensource@toss.im](mailto:opensource@toss.im).

Comment (raon0211) [inline SECURITY.md:21]:
```suggestion
We consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.

If you discover a security vulnerability, report the security issue to the project maintainers directly at [opensource@toss.im](mailto:opensource@toss.im).
```

**Review by raon0211 — DISMISSED** [review]

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+The following table describes the versions of this project that are currently supported with security updates:
+
+| Version | Supported          |
+| ------- | ------------------ |
+| 1.x   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.
```
Comment (raon0211) [inline SECURITY.md:13]:
```suggestion
We consider the security of our systems a top priority. But no matter how much effort we put into system security, there can still be vulnerabilities present.

Please refrain from reporting security vulnerabilities through public GitHub issues, discussions, or pull requests.
```

**Review by raon0211 — APPROVED** [review]

**Pull request merged**